MAUI/ImageEditor/Effects.md

```diff
@@ -0,0 +1,155 @@
+
+* Brightness
+* Blur
+* Contrast
+* Exposure
+* Hue
+* Saturation
+* Sharpen 
+* Opacity
+* None
+
+The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.
+
+## Brightness
+
+Brightness is used to adjusts the overall lightness or darkness of the image. The value of brightness effect ranges from -1 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Brightness, -0.6);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Blur
+
+Blur is used to Creates a soft and unfocused appearance by reducing the image's sharpness. The value of blur effect ranges from 0 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Blur, 0.5);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Contrast
+
+Contrast is used to increases or decreases the difference between light and dark areas, making the image more visually distinct. The value of contrast effect ranges from -1 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect. Contrast, -0.8);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Exposure
+
+Exposure is used to alters the overall brightness and darkness levels of the image. The value of the exposure effect ranges from -1 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect. Exposure, -0.4);
+    . . .
+}
+
+{% endhighlight %}
+
+
+
+## Hue
+
+Hue is used to changes the overall color tone of the image by shifting the color spectrum. The value of hue effect ranges from -1 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Hue, 0.2);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Saturation
+
+Saturation is used to enhances or reduces the intensity and vividness of colors in the image. The value of the saturation effect ranges from -1 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect. Saturation, -0.8);
+    . . .
+}
+
+{% endhighlight %}
+
+
+
+
+## Sharpen
+
+Sharpen is used to enhances the clarity and definition of edges and details in the image. The value of sharpen effect ranges from 0 to 6.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Sharpen, 0.5);
+    . . .
+}
+
+{% endhighlight %}
+
+## Opacity
+
+Opacity is used to controls the transparency or visibility of the image. The value of opacity effect ranges from 0 to 1.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect. Opacity, 0.5);
+    . . .
+}
+
+{% endhighlight %}
+

```

Comment:
Add topic

Save or Cancel applied effects

You should call SaveEdits method to save the applied effects in view, otherwise the effects will be reset on next action.

//// Add code

The applied effects can be cancelled using CancelEdits method or calling ImageEffect method with ImageEffet.None

/// Add code for CancelEdits and None

Reply:
topic added